react-app/README.md

```diff
@@ -57,18 +57,18 @@ So the name you provide to the `ROMCAL_ALIAS` environment var needs to be what i
 
 ### More fun testing bits
 
-If you run the app with `REACT_APP_DAY_VARIANT` set to `developer` or `simple`, the app will appear differently.
+If you run the app with `VITE_DAY_VARIANT` set to `developer` or `simple`, the app will appear differently.
```

Comment:
that's not how vite feeds environment variables in automatically. see [[src](https://vitejs.dev/guide/env-and-mode#env-files)]

> To prevent accidentally leaking env variables to the client, only variables prefixed with `VITE_` are exposed to your Vite-processed code. e.g. for the following env variables: [...]

Reply:
Oh, okay, I haven’t used Vite much, thus I didn’t know about this. Thanks for sharing this with me. (I mostly use Nest.js with `@nestjs/config` where I can use whatever env var names I want.)

Reply:
indeed, but we're talking about a frontend vs a server when discussing vite (frontend) vs nestjs (server).

with a frontend, you want to avoid accidentally compiling in all of your local environment variables, which is what this intends to do. similar practices are used on `REACT_APP_` (see previous state) and [nuxtjs](https://nuxt.com/docs/guide/going-further/runtime-config#environment-variables)